tests/braille-specs/vi.yaml

```diff
@@ -25,32 +24,23 @@ table:
 flags: {testmode: bothDirections}
 tests:
   # digits
-  - [1, ⠼⠁]
-  - [2, ⠼⠃]
-  - [3, ⠼⠉]
-  - [4, ⠼⠙]
-  - [5, ⠼⠑]
-  - [6, ⠼⠋]
-  - [7, ⠼⠛]
-  - [8, ⠼⠓]
-  - [9, ⠼⠊]
-  - [0, ⠼⠚]
-  - [123, ⠼⠁⠃⠉]
-  - [1.234, ⠼⠁⠲⠃⠉⠙]
-  - ["1,234", ⠼⠁⠂⠃⠉⠙]
-  - [1AB, ⠼⠁⠸⠁⠃]
+  - [1 2 3 4 5 6 7 8 9 0, ⠼⠁ ⠼⠃ ⠼⠉ ⠼⠙ ⠼⠑ ⠼⠋ ⠼⠛ ⠼⠓ ⠼⠊ ⠼]
+  - [1123 1.234,56, ⠼⠁⠃⠉ ⠼⠁⠄⠃⠉⠙⠲⠑⠋]
+  - [1AB 2Cd 3cD, ⠼⠁⠸⠁⠃ ⠼⠃⠨⠉⠙ ⠼⠉⠠⠉⠨⠙]
   - [1ab, ⠼⠁⠠⠁⠃, xfail: "backward translation fails"]
-  - [1+2, ⠼⠁⠐⠖⠼⠃]
-  - [2-1, ⠼⠃⠤⠼⠁]
-  - [2*3, ⠼⠃⠐⠦⠼⠉]
-  - [4/5, ⠼⠙⠌⠼⠑]
-  - [3+2(10-5)/6 = 4.16, ⠼⠉⠐⠖⠼⠃⠈⠣⠼⠁⠚⠤⠼⠑⠈⠜⠌⠼⠋ ⠐⠶ ⠼⠙⠲⠁⠋]
+  - [1+2 4-3 5*6 7/8 〉 Ax+B=0, ⠼⠁⠐⠖⠼⠃ ⠼⠙⠤⠼⠉ ⠼⠑⠦⠼⠋ ⠼⠛⠌⠼⠓ ⠐⠕ ⠨⠁⠭⠐⠖⠨⠃⠐⠶⠼⠚]
+  - [3+2(10-5)/6 = 4.16, ⠼⠉⠐⠖⠼⠃⠈⠣⠼⠁⠚⠤⠼⠑⠈⠜⠌⠼⠋ ⠐⠶ ⠼⠙⠄⠁⠋]
+  - [α β γ δ Α Β Γ Δ αβ ΓΔ, ⠰⠁ ⠰⠃ ⠰⠛ ⠰⠙ ⠸⠁ ⠸⠃ ⠸⠛ ⠸⠙ ⠰⠁⠰⠃ ⠸⠛⠸⠙, xfail: "backward translation fails"]
```

Comment:
It is better to put such tests in a separate section that is run only in forward direction.